plugins/prow-job/skills/prow-job-list-qe-jobs/filter_jobs.sh

```
@@ -0,0 +1,69 @@
+
+# If no keywords provided, copy all jobs
+jsonStr=$(sed 's/^var allBuilds = //' "$INPUT_FILE" | sed 's/.$//')
+
+if [ $# -eq 0 ]; then
+    echo "No keywords provided - copying all jobs..."
+    echo "$jsonStr" | jq '{items: .items}' > "$OUTPUT_FILE"
+else
+    # Build the jq filter condition dynamically
+    echo "Looking for jobs containing ALL of the following keywords:"
+    JQ_FILTER=""
+    for keyword in $@; do
```

Shellcheck (0.11.0)

[error] Double quote array expansions to avoid re-splitting elements.

(SC2068)